Exam/CC1/src/exercice4/Reservation.java

```diff
@@ -0,0 +1,41 @@
+package exercice4;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Reservation {
+	private Date date;
+	private int num;
+	
+	public Reservation(int num, String date) throws ParseException {
```

Comment:
Hmmm... Avoid when possible to have constructor that throw.
The prototype should be
~~~java
public Reservation(int num, Date date) {}
~~~
It is the responsibility of the caller to provide a correct date